scoville/command.py

```diff
@@ -94,9 +94,12 @@ def info(mvt_file, kind, d3_json):
     Alternatively, set --d3-json to dump a file suitable for using in D3's
     treemap visualisation.
     """
+    _info(mvt_file, kind, d3_json)
 
+
+def _info(mvt_file, kind, d3_json):
```

Comment:
you can't call @click.command functions directly, so turned the info command into a wrapper that calls _info, so that I could use _info from the compare command as well